src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1236,15 +1236,15 @@ private void duplicateCurrentlySelectedModElement() {
 			ModElement mu = (ModElement) list.getSelectedValue();
 			if (mcreator.getModElementManager().hasModElementGeneratableElement(mu)) {
 				String modName = VOptionPane.showInputDialog(mcreator,
-						"<html><font style=\"font-size: 13px;\">Enter the name of the new mod element:</font><br><small>"
-								+ "This mod element will be the same as " + mu.getName()
-								+ ", but with this name.</font>", "Duplicate " + mu.getName(), mu.getElementIcon(),
+						L10N.t("workspace.elements.duplicate_message1")

```

Comment:
This should be one translation key, no reason to have 3

Reply:
The  `+ mu.getName() +` in middle?

Reply:
![image](https://user-images.githubusercontent.com/16374228/123516497-9ea83c80-d69c-11eb-8498-e3df379f2b30.png)

All this, use replace strings such as {0}

Reply:
?

Reply:
Do not use + + to add data to translation keys, but use the system we prepared for this

Reply:
Example: https://github.com/MCreator/MCreator/blob/508550ba4722b2b800072190188bb016e396f3d0/plugins/mcreator-localization/lang/texts.properties#L297, look into code how variables are filled